packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -158,14 +162,18 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
     if (_state != _DragState.accepted) {
       _state = _DragState.accepted;
       final Offset delta = _pendingDragOffset;
+      final Duration timestamp = _lastPendingEventTimestamp;
       _pendingDragOffset = Offset.zero;
+      _lastPendingEventTimestamp = Duration.ZERO;
```

Comment:
why zero, rather than, e.g., null? zero means something specific, but it's not clear that that's what you mean here

Reply:
in fact you seem to use null everywhere else

Reply:
done